MCprep_addon/materials/generate.py

```diff
@@ -1145,9 +1153,9 @@ def texgen_seus(mat: Material, passes: Dict[str, Image], nodeInputs: List, use_r
 	saturateMixOut = get_node_socket(nodeSaturateMix, is_input=False)
 
 	# Links the nodes to the reroute nodes.
-	RED = "R" if util.min_bv((5, 0)) else "Red"
-	BLUE = "B" if util.min_bv((5, 0)) else "Blue"
-	GREEN = "G" if util.min_bv((5, 0)) else "Green"
+	RED = "Red" if (3, 3) < bpy.app.version < (5, 0) else "R"
+	BLUE = "Blue" if (3, 3) < bpy.app.version < (5, 0) else "B"
+	GREEN = "Green" if (3, 3) < bpy.app.version < (5, 0) else "G"
```

Comment:
Might be a good time to create some sort of `in_bv_range` function in `util.py`, given both the above conditional and this section. Something like this:

```py
def in_bv_range(lower_bound: Tuple, upper_bound: Tuple, inclusive: bool = True) -> bool:
    return min_bv(lower_bound, inclusive) and not min_bv(upper_bound, inclusive)
```